Assets/GLTF/Scripts/GLTFLoader.cs

```diff
@@ -406,7 +407,7 @@ private UnityEngine.Material CreateMaterial(Material def, bool useVertexColors)
 			return material;
 		}
 
-		private const string Base64StringInitializer = "data:application/octet-stream;base64,";
+		private const string Base64StringInitializer = "data:[a-z-]+/[a-z-]+;base64,";
```

Comment:
Force this to only match the start of the string, for performance reasons. `^data:`